zcash_client_backend/src/data_api.rs

```diff
@@ -527,42 +568,16 @@ impl<AccountId: Eq + Hash> WalletSummary<AccountId> {
     /// general usability, including the ability to spend existing funds that were
     /// previously spendable.
     ///
-    /// The window over which progress is computed spans from the wallet's recovery height
-    /// to the current chain tip. This may be adjusted in future updates to better match
-    /// the intended semantics.
-    ///
-    /// Progress is represented in terms of the ratio between notes scanned and the total
-    /// number of notes added to the chain in the relevant window. This ratio should only
-    /// be used to compute progress percentages, and the numerator and denominator should
-    /// not be treated as authoritative note counts. The denominator of this ratio is
-    /// guaranteed to be nonzero.
+    /// The window over which progress is computed spans from the wallet's birthday to the current
+    /// chain tip. It is divided into two segments: a "recovery" segment, between the wallet
+    /// birthday and the recovery height (the height at which recovery from seed was initiated),
```

Comment:
This parenthetical is the thing I want to change in future, and that I was carefuly to convey with the previous language ("This may be adjusted in future updates to better match the intended semantics"). Please retain the previous language in some way in the API documentation (ideally both here and on `Progress`), so that this does not come as a surprise to downstream users later.

Reply:
Ah right, will do.